modules/core/src/main/java/org/apache/fluo/core/async/SyncCommitObserver.java

```diff
@@ -28,33 +30,44 @@
   private volatile boolean aacked = false;
   private volatile Exception error = null;
   private volatile String commitFailMsg = "";
+  private static AtomicBoolean isMethodCalled = new AtomicBoolean(false);
```

Comment:
This should not be static.

Reply:
One really good thing came out of this being static.  While debugging it, I discovered that the async commit code could eat exceptions.  I opened #1014 to fix this.